File: src/URI/URI_1080.java
package URI;
import java.util.Scanner;

public class URI_1080 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int maior = sc.nextInt();
		int posicao = 1;
		
		for(int i = 2; i <=100; i++) {
			int n = sc.nextInt();
			if(n > maior) {
				maior = n;
				posicao = i;
			}
		}
		System.out.println(maior);
		System.out.println(posicao);
		
		sc.close();

	}

}

File: kotlin/URI_1080.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var maior = Int.MIN_VALUE
    var posicao = 0
    repeat(100){
        val n = sc.nextInt()
        if (n > maior) {
            maior = n
            posicao = it + 1
        }
    }
    println("$maior\n$posicao")
}
